kornia/augmentation/_2d/geometric/center_crop.py

```diff
@@ -89,7 +87,7 @@ def __init__(
             padding_mode="zeros",
         )
 
-    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, Tensor]:
+    def generate_parameters(self, batch_shape: Tuple[int, ...]) -> Dict[str, Tensor]:
```

Comment:
No future annotations here ?

Reply:
I'm trying to not drop JIT here, just completing the incomplete defs 😄